fossevents/pages/templates/pages/event_detail.html

```diff
@@ -1,22 +1,33 @@
 {% extends "base.html" %}
 {% load django_markdown %}
-{% block body_classes %}{{block.super}}page--event-detail{% endblock body_classes %}
-{% block title %}{{ event.name }} • {{block.super}}{% endblock title %}
```

Comment:
These two lines were clashing somewhere with the code and removing these two lines of code was making code run perfectly.